web/web.js

```diff
@@ -11,4 +12,35 @@ const app = express();
 const __dirname = new URL(".", import.meta.url).pathname;
 app.use(express.static(__dirname + "/static"));
 
+// REST APIs
+const api = express.Router();
+/**
+ * /wos/users:
+ *      get:
+ *          list of 10 users with most debt and more than -5$ with the following format:
+ *           [
+ *             {
+ *               "id": 2,
+ *               "username": "visitday",
+ *               "balance": 0
+ *             },
+ *             {
+ *               "id": 1,
+ *               "username": "bob",
+ *               "balance": 1337
+ *             }
+ *           ]
+ */
+api.get("/wos/users", async (req, res) => {
+  // first 10 users with a debt superior to $5
+  let users = await user_info()
+    .where("balance", "<", -5)
+    .orWhere("balance", -5)
```

Comment:
balance is measured in cents and we can combine the where clauses so this should be:

```js
.where('balance', '<=', -500)
```